synapse/cortex.py

```diff
@@ -5833,26 +5831,27 @@ async def editCronJob(self, iden, name, valu):
         if name == 'creator':
             await self.auth.reqUser(valu)
             appt.creator = valu
-            await appt._save()
-
-            cdef = appt.pack()
-            await self.feedBeholder('cron:edit:creator', {'iden': iden, 'creator': cdef.get('creator')}, gates=[iden])
-            return cdef
-
-        if name == 'name':
-            await appt.setName(str(valu))
-            pckd = appt.pack()
-            await self.feedBeholder('cron:edit:name', {'iden': iden, 'name': pckd.get('name')}, gates=[iden])
-            return pckd
-
-        if name == 'doc':
-            await appt.setDoc(str(valu))
-            pckd = appt.pack()
-            await self.feedBeholder('cron:edit:doc', {'iden': iden, 'doc': pckd.get('doc')}, gates=[iden])
-            return pckd
-
-        mesg = f'editCronJob name {name} is not supported for editing.'
-        raise s_exc.BadArg(mesg=mesg)
+
+        elif name == 'name':
+            appt.name = str(valu)
+
+        elif name == 'doc':
+            appt.doc = str(valu)
+
+        else:
+            mesg = f'editCronJob name {name} is not supported for editing.'
+            raise s_exc.BadArg(mesg=mesg)
+
+        await appt.save()
+
+        pckd = appt.pack()
+        await self.feedBeholder(f'cron:edit:{name}', {'iden': iden, name: pckd.get(name)}, gates=[iden])
+        return pckd
+
+    @s_nexus.Pusher.onPushAuto('cron:batch')
```

Comment:
lets make this nexus event `cron:edit` and make the method `editCronJob()` :+1:

Reply:
there's already a `editCronJob()` method that uses `cron:edit` as the nexus key: https://github.com/vertexproject/synapse/blob/blackout/SYN-6353/cron-state/synapse/cortex.py#L5824